libs/metrics/test_positivity.py

```diff
@@ -119,7 +119,7 @@ def _make_output_dataset(
     # Drop all-NA timeseries now, as done in from_timeseries_wide_dates_df. This makes sure
     # `locations` is used to build provenance information for only timeseries in the returned
     # MultiRegionDataset.
-    wide_date_df = wide_date_df.dropna("rows", "all")
```

Comment:
use keyword arguments to fix: 

```
FutureWarning: In a future version of pandas all arguments of DataFrame.dropna will be keyword-only
```

I don't actually know what "rows" is here--`axis` in `dropna()` only has the options `index` or `column`. I assume this is the same as `index` because that's what's used to drop rows (and it is the default) so I've replaced it.